packages/rancher-pushprox/charts/templates/_helpers.tpl

```diff
@@ -82,7 +82,6 @@ k8s-app: {{ template "pushProxy.proxy.name" . }}
 
 {{- define "pushProxy.serviceMonitor.labels" -}}
 app: {{ template "pushprox.serviceMonitor.name" . }}
-release: {{ .Release.Name | quote }}
```

Comment:
yes, I am sure the label will not be missing because the helper function includes `pushProxy.commonLabels` in its next line 😆 

```
{{- define "pushProxy.serviceMonitor.labels" -}}
app: {{ template "pushprox.serviceMonitor.name" . }}
release: {{ .Release.Name | quote }}      <------- removed by this PR 
{{ template "pushProxy.commonLabels" . }} <------- use pushProxy.commonLabels
{{- end -}}
```